recipes/oscars/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "oscars" %}
+{% set version = "2.1.8" %}
+{% set sha256 = "c37853089db601537c48484ee72b84ae40893daf0d4b86030e4bf080dc3965b7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dhidas/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [unix]
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - matplotlib
+    - mpi4py  # [not win]
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - oscars.sr
```

Comment:
Might want to consider some more import tests. `setup.py` lists Windows build files, but you're not asking for a compiler on Windows--I'd expect the package to be broken.

Reply:
I like the idea. I think all entries from https://github.com/dhidas/OSCARS/blob/1f0823c3a8c9637b4c28e0afffefd361a62d8136/oscars/__init__.py#L1 can be added to the import test block.